src/components/error-boundary.jsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import logo from '../../catwalk.svg';
```

Comment:
We should have the logo in the `src/` folder imo, the web app shouldn't pull resources from outside its root folder.